include/arkode/arkode_mristep.h

```diff
@@ -62,15 +64,27 @@ typedef enum
   ARKODE_IMEX_MRI_GARK_EULER,
   ARKODE_IMEX_MRI_GARK_TRAPEZOIDAL,
   ARKODE_IMEX_MRI_GARK_MIDPOINT,
-  ARKODE_MAX_MRI_NUM = ARKODE_IMEX_MRI_GARK_MIDPOINT,
+  ARKODE_MERK21,
+  ARKODE_MERK32,
+  ARKODE_MERK43,
+  ARKODE_MERK54,
+  ARKODE_IMEX_MRI_SR21,
+  ARKODE_IMEX_MRI_SR32,
+  ARKODE_IMEX_MRI_SR43,
+  ARKODE_MAX_MRI_NUM = ARKODE_IMEX_MRI_SR43
 } ARKODE_MRITableID;
 
-/* Default MRI coupling tables for each order */
+/* Default MRI coupling tables for each order and type */
 static const int MRISTEP_DEFAULT_EXPL_1 = ARKODE_MRI_GARK_FORWARD_EULER;
 static const int MRISTEP_DEFAULT_EXPL_2 = ARKODE_MRI_GARK_ERK22b;
 static const int MRISTEP_DEFAULT_EXPL_3 = ARKODE_MIS_KW3;
 static const int MRISTEP_DEFAULT_EXPL_4 = ARKODE_MRI_GARK_ERK45a;
 
+static const int MRISTEP_DEFAULT_EXPL_2_AD = ARKODE_MRI_GARK_ERK22a;
```

Comment:
I'd left `ARKODE_MRI_GARK_ERK22b` as the default second-order explicit method for backwards compatibility, but when adding temporal adaptivity I needed an adaptive method for the adaptive default.  Either I couldn't figure out the embedding for `ARKODE_MRI_GARK_ERK22b`, or I just couldn't get the one in Adrian's paper to work, necessitating the choice for `ARKODE_MRI_GARK_ERK22a`.

While I agree with your comment elsewhere that it would be nice for us to have the same default methods for adaptive and non-adaptive runs, many of our previous defaults have no embeddings, and thus cannot serve as adaptive defaults.  Furthermore, some of our fixed-step defaults require less work-per-step than the adaptive methods at the same order.  I chose the current set to provide consistency with previous releases, and to ensure that we do not force unnecessarily expensive fixed-step defaults.

Reply:
Looking at Adrian's paper (bottom of page 2316), the `ARKODE_MRI_GARK_ERK22b` method ($c_2=1$) would mean the embedded gamma coefficients are all 0 and thus $y_n = \widehat{y}_n$. Something seems off here and that might explain why that didn't work for you. Perhaps something I can look at Monday. If we could get `ARKODE_MRI_GARK_ERK22b` working, it would be very efficient since the embedding doesn't require an additional integration.

Reply:
Are you sure about that (not requiring an additional integration).  Even if the forcing terms for the embedding are all zero, it would still require integration over the interval [c_{s-1},1] using a *different* forcing function than was used for the final internal stage.

Reply:
I'm sorry, let's rewind this conversation a bit.  I was just looking in the code to test what had gone wrong with the 22b embedding, and it looks like I'd fixed it.  I must have made that fix after setting the default tables.

Given that 22b was the default fixed-step method before, I'll change it to also be the default adaptive step method.